modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,50 @@
+
+By moving from an "in-payload" image stream to an image stream that references non-payload images, {product-title} can define as many image stream tags as needed. Red Hat has created a series of new image stream tags to go along with the existing `"value": "jenkins:2"` and `"value": "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base:latest"` image stream tags present in {product-title} 4.10 and earlier. These new image stream tags address some requests to improve how the Jenkins-related image streams are maintained.
+
+About the new image stream tags:
+
+`ocp-upgrade-redeploy`:: To update your Jenkins image when you upgrade {product-title}, you use this image stream tag in your Jenkins deployment configuration. This image stream tag corresponds to the existing `2` image stream tag of the `jenkins` image stream and the `latest` image stream tag of the `jenkins-agent-base` image stream. It employs an image tag specific to only one SHA or image digest. When changes of note are made to the `ocp-tools-4` image, such as for Jenkins security advisories, Red Hat Engineering updates the Cluster Samples Operator payload.
+
+`user-maintained-upgrade-redeploy`:: To manually redeploy Jenkins after you upgrade {product-title}, you use this image stream tag in your Jenkins deployment configuration. This image stream tag uses the least specific image version indicator available. When you redeploy Jenkins, you run the following command: `$ oc import-image jenkins:user-maintained-upgrade-redeploy -n openshift`. When you issue this command, the {product-title} `ImageStream` controller accesses the `registry.redhat.io` image registry and stores any updated images in the {product-title} internal image registry's slot for that Jenkins `ImageStreamTag` object. Otherwise, if you do not run this command, your Jenkins deployment configuration does not trigger a redeployment.
+
+`scheduled-upgrade-redeploy`:: To automatically redeploy the latest version of the Jenkins image when it is released, you use this image stream tag in your Jenkins deployment configuration. This image stream tag uses the periodic importing of image stream tags feature of the {product-title} image stream controller, which checks for changes in the backing image. If the image changes, for example, when we push changes due to a recent Jenkins security advisory, {product-title} triggers a redeployment of your Jenkins deployment configuration. See "Configuring periodic importing of image stream tags" in the following "Additional resources."
+
+
+.What happens with the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams in the `openshift` namespace?
+
+As mentioned, the associated in-payload `OpenShift Jenkins Maven` and `NodeJS Agent` images for {product-title} are deprecated in 4.11, no longer included in the OpenShift install payload. They do not have alternatives defined in the `ocp-tools-4` repository.
```

Comment:
We should remind that even if there is no direct equivalent, the other way to work this out is to use what we "sidecar" like pattern and give a link to it in the existing documentation.

Reply:
Updated. Thanks.